bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/EventCondition.cpp

```diff
@@ -141,7 +141,7 @@ inline void OpenSpaceToolkitAstrodynamicsPy_EventCondition(pybind11::module& aMo
                 R"doc(
                     The value of the target.
 
-                    :type: Real
+                    :type: float
```

Comment:
Are you sure it gets returned as a float? Don't we have to cast it in python to make sure its a float?

Reply:
It's technically a Real you're right. But I think it's clearer in the docs to say float as they can be used almost interchangably (except during JSON serialization) and that way users don't have to check what a `Real` is.